Removing @expectedFailureXLA from test_nll_loss_empty_tensor_reduction_mean

pbelevich opened the pull request:
Because it's disabled in XLA(https://github.com/pytorch/xla/pull/1563)
Discussed in https://github.com/pytorch/xla/issues/1539

Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#32701 Removing @expectedFailureXLA from test_nll_loss_empty_tensor_reduction_mean**

Differential Revision: [D19633349](https://our.internmc.facebook.com/intern/diff/D19633349)

pbelevich committed:
Removing @expectedFailureXLA from test_nll_loss_empty_tensor_reduction_mean

[ghstack-poisoned]

zou3519 commented on test/test_nn.py:
```diff
@@ -10647,7 +10647,6 @@ def test_nll_loss_empty_tensor_reduction_none(self, device):
         self._nll_loss_helper([2, 3, 5, 7, 0], "none", torch.empty([2, 5, 7, 0], device=device), device)
 
     @unittest.skipIf(TEST_WITH_UBSAN, "division-by-zero error with UBSAN")
-    @expectedFailureXLA  # https://github.com/pytorch/xla/issues/1539
```
Can you also delete the implementation of the expectedFailureXLA decorator because IIRC it's not used anywhere else?

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit 9209c8de:


* **1/1** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=9209c8de45529fbf22b678bd065e593f563103d2).

### :detective: 1 new failure recognized by patterns
The following build failures do not appear to be due to upstream breakage:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4359263) pytorch_macos_10_13_py3_test (1/1)
**Step:** "Test" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=92014183) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=92014183))
<details>
<summary>
<code>Jan 28 07:54:59 RuntimeError: test_autograd failed!</code>
</summary>

```
Jan 28 07:54:58 Ran 1046 tests in 280.512s 
Jan 28 07:54:58  
Jan 28 07:54:58 FAILED (errors=1, skipped=13, expected failures=1) 
Jan 28 07:54:58  
Jan 28 07:54:58 Generating XML reports... 
Jan 28 07:54:59 Traceback (most recent call last): 
Jan 28 07:54:59   File "test/run_test.py", line 456, in <module> 
Jan 28 07:54:59     main() 
Jan 28 07:54:59   File "test/run_test.py", line 449, in main 
Jan 28 07:54:59     raise RuntimeError(message) 
Jan 28 07:54:59 RuntimeError: test_autograd failed! 
Jan 28 07:54:59 + cleanup 
Jan 28 07:54:59 + retcode=1 
Jan 28 07:54:59 + set +x 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

pbelevich committed:
Update on "Removing @expectedFailureXLA from test_nll_loss_empty_tensor_reduction_mean"

Because it's disabled in XLA(https://github.com/pytorch/xla/pull/1563)
Discussed in https://github.com/pytorch/xla/issues/1539




[ghstack-poisoned]
